src/core/qgsmaprenderersequentialjob.cpp

```diff
@@ -31,6 +31,7 @@ QgsMapRendererSequentialJob::QgsMapRendererSequentialJob( const QgsMapSettings&
   mImage = QImage( mSettings.outputSize(), mSettings.outputImageFormat() );
   mImage.setDotsPerMeterX( 1000 * settings.outputDpi() / 25.4 );
   mImage.setDotsPerMeterY( 1000 * settings.outputDpi() / 25.4 );
+  mImage.fill( settings.backgroundColor().rgba() );
```

Comment:
According to the QImage documentation : 
> **QImage::QImage(const QSize & size, Format format)**
Warning: This will create a QImage with uninitialized data. Call fill() to fill the image with an appropriate pixel value before drawing onto it with QPainter
Line 70, it's a QPainter object. It's leaving the original noise.

I was trying to get transparent tiles with this python script from @wonder-sk : https://github.com/wonder-sk/qgis-tile-server I got some problem about noise when I tried with transparency. The script works very well with opaque background. Martin gave me a hint : https://github.com/wonder-sk/qgis-tile-server/issues/3#issuecomment-178675826